src/test/suite/fixtures/recorded/languages/latex/changeName.yml

```diff
@@ -0,0 +1,25 @@
+languageId: latex
```

Comment:
Nice.  As mentioned above, would be cool if this one worked from within the section itself as well